daprdocs/content/en/reference/components-reference/supported-middleware/middleware-wasm.md

```diff
@@ -32,15 +40,15 @@ spec:
 
 ## Spec metadata fields
 
-| Field | Details | Example |
-|-------|---------|---------|
-| path | A relative or absolute path to the WASM module | "./hello.wasm" |
-| runtime | The WASM runtime of your WASM binary. Only `wazero` is supported | ["wazero"](https://github.com/tetratelabs/wazero) |
-
+| Field    | Details                                        | Example        |
+|----------|------------------------------------------------|----------------|
+| path     | A relative or absolute path to the Wasm binary | "./hello.wasm" |
+| poolSize | The instance count of the Wasm binary          | 10             |
```

Comment:
Need to say whether `poolsize is optional or required`. And describe a little more on what this does. Does it instantiate this number of Wasm binary images?

Reply:
> Need to say whether poolsize is optional or required. And describe a little more on what this does. Does it instantiate this number of Wasm binary images?

I will add text that it is optional. WRT how to describe it, the pool size is the amount of instances of the binary, specified by path. instance and binary is to wasm as container and image is to docker. Advice on rewording?

Reply:
I pitched a rewording. Let me know if it needs more clarity